cmd/kubectl-gadget/biolatency.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
+	gadgetv1alpha1 "github.com/kinvolk/inspektor-gadget/pkg/api/v1alpha1"
+)
+
+const (
+	GADGET_NAME        = "biolatency"
+	GADGET_OUTPUTMODE  = "Status"
+	GADGET_OUTPUTSTATE = "Completed"
+)
+
+var biolatencyTraceConfig = &utils.TraceConfig{
+	GadgetName:                GADGET_NAME,
+	TraceOutputMode:           GADGET_OUTPUTMODE,
+	TraceOutputState:          GADGET_OUTPUTSTATE,
+	CommonFlags:               &params,
+	IgnoreKubeConfigNamespace: true,
+}
+
+var biolatencyLogger *log.Entry
+
+var biolatencyCmd = &cobra.Command{
+	Use:   "biolatency",
+	Short: "Generate a histogram with the distribution of block device I/O latency",
+}
+
+var biolatencyStartCmd = &cobra.Command{
+	Use:    "start",
```

Comment:
We should decide if we use `start/stop` or `monitor/report`.
I am OK with both, but we need to have the same CLI subcommands.

Reply:
Indeed! In fact, the idea is to use `star/stop` for all the gadgets.

Reply:
Thank you for clarifying, I will then switch `seccomp-advisor` `start`/`stop`.